include/NovelRT.Interop/Animation/NrtAnimationTypedefs.h

```diff
@@ -1,19 +1,21 @@
-// Copyright © Matt Jones and Contributors. Licensed under the MIT License (MIT). See LICENCE.md in the repository root for more information.
+// Copyright © Matt Jones and Contributors. Licensed under the MIT License (MIT). See LICENCE.md in the repository root
+// for more information.
```

Comment:
Is this intentional?

Reply:
Yes - MS-style clang-format sets a column limit of 120, which works rather well with our parameter changes (it helps keep each param on one line when breaking them apart like we have).

On that note, it also appears to not effect Doxy in a negative way so we should be alright with this.